src/MCPClient/lib/clientScripts/archivematicaClamscan.py

```diff
@@ -94,7 +94,11 @@ def main(fileUUID, target, date):
         record_event(fileUUID, version, date, outcome='Fail')
         return 1
 
-    state, details = result[target]
+    try:
+        state, details = result[target]
+    except KeyError:
+        logger.error('File not scanned: %s', target)
+        return 1
```

Comment:
There is no behavior changes here, I'm just protecting us from the scenario where the result dict doesn't have a member `target` which frequently means that `clamavd` failed to scan the file (e.g. permission issues) and it did not report virus scanning results on it.